src/util/errors/ConflictHttpError.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpError } from './HttpError';
```

Comment:
Might want all of those in one file? @joachimvh?

Reply:
I think we originally decided it was cleaner to have every class in its own separate file.

Reply:
Arguments against that would be:
- in one file, it is easier to apply structure (for instance, section/order them by HTTP status code)
- they will all be very short files anyway (so a lot of breadth for virtually no depth)

What would arguments for be?